src/hooks/useReactions.ts

```diff
@@ -1,41 +1,74 @@
 import { useEffect, useState } from "react";
 import { useFirebase } from "react-redux-firebase";
+
 import { REACTION_TIMEOUT } from "settings";
-import { Reaction } from "utils/reactions";
+
+import { isReactionCreatedBy, Reaction } from "types/reactions";
+import { User } from "types/User";
+
+import { WithId } from "utils/id";
+
+export interface UseReactionsProps {
+  venueId?: string;
+  user?: WithId<User>;
+}
 
 // @debt refactor this to use useConnect like in src/components/templates/ReactionPage/ReactionPage.tsx ?
-export const useReactions = (venueId?: string) => {
+export const useReactions = ({ venueId, user }: UseReactionsProps) => {
   const [reactions, setReactions] = useState<Reaction[]>([]);
   const firebase = useFirebase();
 
+  const userId = user?.id;
```

Comment:
I guess we only use userId here? Then, it makes sense to only pass id

Reply:
As the prop? Yeah, that makes sense. I only just changed the code to extract this last night, before that I think I assumed I was going to be needing more than just the `userId` here.

Reply:
fixed in https://github.com/sparkletown/sparkle/commit/119bfae9a3770b7a2f8f14f28b06fcb212f55d97